### PR TITLE
Update to QEMU v9.1.1

### PR DESCRIPTION
**Pull request by rmalmain**

### CODE REVIEW

File: accel/tcg/tcg-runtime-sym-common.c [hunk 1]
```diff
@@ -22,31 +40,42 @@ void *build_and_push_path_constraint(CPUArchState *env, void *arg1_expr, void *a
...
+            handler = _sym_build_tstne;
+            break;
+
         case TCG_COND_LT:
             handler = _sym_build_signed_less_than;
             break;
         case TCG_COND_GE:
             handler = _sym_build_signed_greater_equal;
             break;
-        case TCG_COND_LE:
```
Comment (aurelf) [inline accel/tcg/tcg-runtime-sym-common.c:31]: Why moving this code ?

File: accel/tcg/tcg-runtime-sym-common.c [hunk 2]
```diff
@@ -22,31 +40,42 @@ void *build_and_push_path_constraint(CPUArchState *env, void *arg1_expr, void *a
...
-            handler = _sym_build_signed_less_equal;
-            break;
         case TCG_COND_GT:
             handler = _sym_build_signed_greater_than;
             break;
+        case TCG_COND_LE:
```
Comment (aurelf) [inline accel/tcg/tcg-runtime-sym-common.c:60]: Same?

File: accel/tcg/tcg-runtime-sym-common.c [hunk 3]
```diff
@@ -22,31 +40,42 @@ void *build_and_push_path_constraint(CPUArchState *env, void *arg1_expr, void *a
...
         case TCG_COND_GEU:
             handler = _sym_build_unsigned_greater_equal;
             break;
-        case TCG_COND_LEU:
-            handler = _sym_build_unsigned_less_equal;
-            break;
         case TCG_COND_GTU:
             handler = _sym_build_unsigned_greater_than;
             break;
+        case TCG_COND_LEU:
```
Comment (aurelf) [inline accel/tcg/tcg-runtime-sym-common.c:73]: Same?

**Review by aurelf — COMMENTED** [review]
Some parts of the patch seem not useful?

**Comment by aurelf** [comment]
OK that makes sense, and this is SymQEMU code, so we can keep this here (no upstream needed). Ideally this could be a separate commit or have an note about it in the commit message.

**Comment by rmalmain** [comment]
i can add this in the final squash's comment, would it be fine that way?

**Pull request merged**